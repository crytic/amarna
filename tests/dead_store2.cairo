func are_equal(x, y) -> (eq: felt) {
    if (x == y) {
        let (a) = 44;
        return (1,);
    } else {
        return (0,);
    }
}
