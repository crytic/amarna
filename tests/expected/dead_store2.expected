[dead-store] in dead_store2.cairo:3:14
[unused-function] in dead_store2.cairo:1:1